docs/conf.py

```diff
@@ -1,47 +1,44 @@
+# Configuration file for Sphinx to build our documentation to HTML.
+#
+# Configuration reference: https://www.sphinx-doc.org/en/master/usage/configuration.html
+#
+from datetime import datetime
+
+
+# -- Project information -----------------------------------------------------
+# ref: https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+#
 project = "JupyterHub Sphinx Theme"
-copyright = "2022"
+copyright = f"2022 - {datetime.now().year}, JupyterHub"
 author = "JupyterHub"
-main_doc = "index"
-version = "0.0.1a"
 
-# -- General configuration ---------------------------------------------------
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
+# -- General Sphinx configuration ---------------------------------------------------
+# ref: https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+#
 extensions = [
     "myst_parser",
+    "jupyterhub_sphinx_theme",
 ]
+root_doc = "index"
+source_suffix = [".rst", ".md"]
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-
-# -- Options for HTML output -------------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
+# -- Options for HTML output ----------------------------------------------
+# ref: http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 #
-html_theme = "jupyterhub_sphinx_theme"
-html_title = "JupyterHub Sphinx Theme"
-html_copy_source = True
-html_sourcelink_suffix = ""
```

Comment:
We don't set these things in other sphinx conf.py files, so I figure we shouldn't here either.
- `html_title` gets a value anyhow from the rendered pages
- `html_copy_source` is defaulting to True anyhow
- `html_sourcelink_suffix` could possibly be relevant to set explicitly to override the default of `.txt`, but I don't know when and why so I'm assuming this is just a copy/paste legacy config of no purpose for this projects function